apps/minifront/src/components/swap/auction-list/end-or-withdraw-all-button.tsx

```diff
@@ -89,26 +93,22 @@ export const EndOrWithdrawAllButton = () => {
         <Tooltip>
           <TooltipTrigger
             onClick={() => {
-              // Chain has a transaction size limit, so we can add at most a batch of 48 auctions in a single transaction
-              // see https://github.com/penumbra-zone/web/issues/1166#issuecomment-2263550249
-              const auctionBatch = assembleAuctionBatch(data, 1n, 48);
+              const auctionBatch = assembleAuctionBatch(data, 1n, BATCH_LIMIT);
               void withdrawAllAuctions(
                 auctionBatch.auctions,
                 // TODO Should use the index of the selected account after the account selector for the auction is implemented
                 auctionBatch.source,
               );
             }}
-            aria-label='Withdraw all ended auctions, with a limit of 48 auctions per transaction'
+            aria-label={withdrawTooltipContent}
           >
-            <div className='w-[95px] shrink-0'>
+            <div className='w-[120px] shrink-0'>
```

Comment:
Making button wider to accommodate larger text length 

<img width="350" alt="Screenshot 2024-08-29 at 3 56 19 PM" src="https://github.com/user-attachments/assets/bdd56521-eef6-4aa4-ae80-44474ab88ae9">